### PR TITLE
Solution of Fractional Knapsack and Modified FractionalKnapsack #133(part1 and part2 Solved)

### PR DESCRIPTION
**Pull request by jstjyoti**
The fractional knapsack problem solved using greedy approach.

### CODE REVIEW

**Comment by jstjyoti** [comment]
@ramitsawhney27  @sahilbansal17 I have completed both the parts (part1- Fractional Knapsack Problem and Part2-Fractional Knapsack of objects with Values) as I was asked and assigned to.
Kindly review and suggest if and changes required in these two codes.

File: Algorithms/Greedy/KnapsackUpdatedWithValuesOfObjects.c [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+		profit+=(ptr[i].p)*x[i]*(ptr[i].v)*x[i]*x[i];
+	}
+	printf("\nTotal profit by keeping objects with values is %f",profit);
+
+}
+/*n objects are inputed(weight,profit of each object).Each object is given an id starting from A .
+The objects are sorted according to profit per unit weight (any sorting algorithm can be used).Here I have used Bubble Sort for Simplicity .
+Time Complexity of total program entirely depends upon the sorting algorithm as the complexity of greedy approach of Fractionalknapsack function is O(n).
+Here,total complexity is O(n^2) as the complexity is 
+O(n^2)+O(n).*/
```
Comment (ramitsawhney27) [inline Algorithms/Greedy/KnapsackUpdatedWithValuesOfObjects.c]: Should change it to a sorting algorithm that works in O(nlogn). Use the STL sort function for the same.

**Comment by jstjyoti** [comment]
@ramitsawhney27  @sahilbansal17  I have made changes As suggested .
Review it Once .

**Comment by jstjyoti** [comment]
I have changed kindly review

**Comment by jstjyoti** [comment]
@ramitsawhney27 review

**Comment by jstjyoti** [comment]
@sahilbansal17 review

**Comment by sahilbansal17** [comment]
@jstjyoti Please delete the heapSort and floydWarshall files since they have been merged eariler.

**Comment by jstjyoti** [comment]
@sahilbansal17  I have made changes kindly review now

**Pull request merged**